product_docs/docs/pgd/5/rel_notes/pgd_5.3.0_rel_notes.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "EDB Postgres Distributed 5.3.0 release notes"
+navTitle: "Version 5.3.0"
+---
+
+Released: 14 Nov 2023
+
+EDB Postgres Distributed version 5.3.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.3.
+!!!
+
+## Highlights of EDB Postgres Distributed 5.3.0
+
+* Postgres 16 and EPAS 16 supported
+
+## Compatibility
+
+PGD 5.3 is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8, and later.
+
+## Features
+
+| Component | Version | Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| PGD | 5.3.0 | Add support for PostgreSQL 16 server. | |
```

Comment:
I don't think we use `PGD` as component. Should this be `BDR` ?

Reply:
Well, does proxy, cli, and utilities not support PG16?

Not meaning to blame anybody, but let's be clear here. Ideally, all of PGD should support PG16 (and derivatives). If there's any issue with any of the components, let's try to address those.

Reply:
My point is `PGD` is the complete product and `BDR`, `CLI`, `Proxy` and `Utilities` are it's components. So putting `PGD` as component doesn't sound correct. I don't think I have seen `PGD` being used as component in release notes in past.
`BDR` is a major component of `PGD` so if `BDR` supports a particular version of PG then it's assumed that the other components do support that version of PG. If any component doesn't then it would be good if we bring that component also onboard.

Reply:
Then maybe "component: all" to clarify that? Implicitly assuming BDR means all components seems even worse, IMO.

Reply:
That would mean that we cannot say the entire product is supporting something and would have to enumerate that support component by component. 

> BDR is a major component of PGD so if BDR supports a particular version of PG then it's assumed that the other components do support that version of PG.

That assumes the reader of the release notes knows that assumption is being used. As far as I know, that assumption has never been explicitly or implicitly called out in documentation or release notes. 

If we switch this to BDR, we will need to add an explicit explanation that the BDR-supported database level is also supported by all other PGD components in this release. 

Or we can use the simple shorthand that PGD refers to all elements of the PGD product.

Reply:
With that we circled back to where we started from. :-)  @nataliawojcik27 prefer "all" or "pgd" here?